### ISSUE TITLE
Buttons do not have accessible names

### ISSUE THREAD

**Issue by vitbokisch** [vitbokisch/bokisch.com#3]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers.
